content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/posttransform.yaml

```diff
@@ -10,49 +10,49 @@ object:
   fields:
   - State2:
       value: "/Transformation/Infrastructure/VM/${state_var#destination_ems_type}/SetDescription"
-      on_entry: /System/CommonMethods/MiqAe.WeightedUpdateStatus(weight => 5, description
-        => "Update description of VM", task_message => "Migrating")
+      on_entry: /Transformation/TransformationThrottler.Watch ; /System/CommonMethods/MiqAe.WeightedUpdateStatus(weight
```

Comment:
stating my interpretation:
`TransformationThrottler.Watch` is basically an added state transformation to ensure there is back pressure on the producer

Reply:
Yes, it checks that the throttler is still there and starts a new one if not. That's a watchdog, hence its code being really tight.